src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -131,12 +138,15 @@
          amount-in               (send-utils/amount-in-hex amount (if token token-decimal 0))
          from-address            wallet-address
          disabled-from-chain-ids []
-         disabled-to-chain-ids   []
+         disabled-to-chain-ids   (if (= transaction-type :bridge)
+                                   (filter #(not= % bridge-to-chain-id) network-chain-ids)
+                                   [])
          from-locked-amount      {}
-         transaction-type        (if token
-                                   constants/send-type-transfer
-                                   constants/send-type-erc-721-transfer)
-         request-params          [transaction-type
+         transaction-type-param  (case transaction-type
+                                   :collectible constants/send-type-erc-721-transfer
+                                   :bridge      constants/send-type-bridge
+                                   constants/send-type-transfer)
```

Comment:
In general I think the approach is fine, but...

I think types should be just `:send` or `:bridge`. We could potentially bridge collectibles in the future, so the code should be flexible enough to support this. A collectible is just a different kind of token, so I am not convinced to have a transaction type of `:collectible`, that could lead to confusion in the future.

If we only have `:send` and `:bridge` transaction types, then checking validity of`[:wallet :ui :send :collectible]` or `[:wallet :ui :send :token]` in app-db would be enough for setting the correct `transaction-type-param` required by status-go endpoint.

cc @J-Son89 @Pau1fitz

Reply:
true but `:bridge` is just a form of sending as well? 
Mostly `transfer-type` here is related to the parameter of the `get-suggested-routes` endpoint. We know in advance depending on the flow of which type we want so why should we leave it up to this function to resolve or figure out which type of transaction it is.
The parameters of this function some what depend on the different send flows too.

If in the future we are bridging a collectible the transaction type would be directly related to the value needed for `get-suggested-routes` endpoint.
I'm happy either way just giving the thought behind this 👍

Reply:
Also, for another issue, but we may consider refactor the db path where we store transaction data from `:wallet :ui :send` to `:wallet :ui :transaction` (or `:wallet :ui :tx` if we want shorter names)

Reply:
I think of bridging as something different of sending.

- Send: is a wallet to wallet transaction
- Bridge: Process of moving user tokens from one network to other networks

Sending may involve bridging (multiple transactions) if the response of getSuggestedRoutes involves a bridging transaction, but at this point in the flow we should just know if it is a send or bridge, and which token we want to send or bridge.

Anyways, as I said, I think that the approach is fine for now, I may propose changes in the future and we can discuss them later on. I am thinking also of changing the path in app-db to be `[:wallet :ui :transaction]` instead of `[:wallet :ui :send]` to make it more flexible, but I need to think it better while we integrate both flows and that may require a bigger refactor in the case we agree to do it.

Reply:
Sounds good @briansztamfater, happy to go with whatever you suggest here 👍